src/components/Header/Header.scss

```diff
@@ -66,15 +64,22 @@ header {
   display: none;
 
   @include for-desktop-up {
-    padding: 0 32px;
-    max-height: 176px;
     display: flex;
-    flex-flow: column wrap;
+    width: 100%;
     justify-content: space-between;
-    margin-bottom: 24px;
   }
 }
 
+.titleContainer,
+.orgContainer {
+  padding: 0 32px;
+  max-height: 176px;
+  display: inline-flex;
```

Comment:
Just an FYI. Since the parent of these two is `display: flex`, these guys are 'flex items', so there is no difference between `inline-flex` and `flex` (so this could just be flex).